Library/Homebrew/github_runner_matrix.rb

```diff
@@ -100,6 +100,9 @@ def create_runner(platform, arch, spec, macos_version = nil)
     runner.freeze
   end
 
+  NEWEST_GITHUB_ACTIONS_MACOS_RUNNER = :ventura
```

Comment:
While perhaps not relevant yet, there can be a oldest too. GitHub deprecated 10.15 a few months before we did.

Reply:
Yup, I thought about that. But also figured that it was not relevant yet.